kotlin-react-router/src/main/generated/react/router/AwaitResolveRenderFunction.kt

```diff
@@ -0,0 +1,12 @@
+@file:JsModule("react-router")
+@file:JsNonModule
+
+@file:Suppress(
+    "NON_EXTERNAL_DECLARATION_IN_INAPPROPRIATE_FILE",
+    "NAME_CONTAINS_ILLEGAL_CHARS",
+)
+
+package react.router
+
+
+typealias AwaitResolveRenderFunction = (data: Any? /* Awaited<any> */) -> react.ReactNode
```

Comment:
`Awaited`?

Reply:
No, [read the docs](https://www.typescriptlang.org/docs/handbook/utility-types.html#awaitedtype)

Reply:
`(data: Awaited<Any>?) -> ReactNode` expected